inc/options/general.php

```diff
@@ -457,6 +457,65 @@
 
 		<?php Util_Ui::postbox_footer(); ?>
 
+		<?php
+		Util_Ui::postbox_header_tabs(
+			esc_html__( 'WP Cron', 'w3-total-cache' ),
+			esc_html__( 'Enabling this will schedule a WP-Cron event that will flush all enabled Caches. If you prefer to use a system cron job instead of WP-Cron, you can schedule the following command to run at your desired interval: "wp w3tc flush all". If the Always Cached extension is active and enabled, page cache entries will instead be added to the queue instead of being purged from the cache.', 'w3-total-cache' ),
+			'',
+			'allcache_wp_cron'
+		);
+		?>
+		<table class="form-table">
+			<?php
+			$c           = Dispatcher::config();
+			$wp_disabled = ! $c->get_boolean( 'allcache.wp_cron' );
+
+			Util_Ui::config_item(
+				array(
+					'key'            => 'allcache.wp_cron',
+					'label'          => esc_html__( 'Enable WP-Cron Event', 'w3-total-cache' ),
+					'checkbox_label' => esc_html__( 'Enable', 'w3-total-cache' ),
+					'control'        => 'checkbox',
+				)
+			);
+
+			$time_options = array();
+			for ( $hour = 0; $hour < 24; $hour++ ) {
+				foreach ( array( '00', '30' ) as $minute ) {
+					$time_value                  = $hour * 60 + intval( $minute );
+					$time_label                  = gmdate( 'g:i a', strtotime( sprintf( '%02d:%s', $hour, $minute ) ) );
```

Comment:
The config values should be in UTC/GMT, but the label in locale time -- use https://developer.wordpress.org/reference/functions/get_date_from_gmt/

Reply:
Updated. Not sure if the changes satisfy